examples/kotlin/ftl-module-ad/src/main/kotlin/ftl/ad/Ad.kt

```diff
@@ -16,12 +19,14 @@ class AdModule {
   private val database: Map<String, Ad> = loadDatabase()
 
   @Verb
-  @Ingress(Method.GET, "/get")
-  fun get(context: Context, req: AdRequest): AdResponse {
-    return when {
-      req.contextKeys != null -> AdResponse(ads = contextualAds(req.contextKeys))
-      else -> AdResponse(ads = randomAds())
+  @Ingress(Method.GET, "/get", HTTP)
```

Comment:
Weird, we should get rid of `HTTP` and call this `@HttpIngress`. It's not going to be usable for any other type such as gRPC anyway.

Reply:
Yeah good call. I'll make a ticket to clean this up.
https://github.com/TBD54566975/ftl/issues/874